### PR TITLE
Added ePIMS ID to venues

### PR DESCRIPTION
**Pull request by JamieHMCTS**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/SSCS-10244

### Change description ###
Added ePIMS ID to list of venues in venues.csv

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ x] No
```

### CODE REVIEW

**Comment by suetlaiHMCTS** [comment]
You may need to post the Jira link in this pr, [https://tools.hmcts.net/jira/browse/SSCS-10244]

**Review by gorkem-turan — APPROVED** [review]

**Review by dattipoeHMCTS — DISMISSED** [review]
Should all these have ePIMS IDs?

**Comment by JamieHMCTS** [comment]
@dattipoeHMCTS correct, all the venues should have ePIMS ID's. However the ones added in this file are the only ones available in our documentation at this time. A further list is going to compiled at a later date.

File: src/main/resources/reference-data/sscs-venues.csv [hunk 1]
```diff
@@ -1,266 +1,266 @@
...
-1257,"SC336","SSCS Bradford","Hastings Law Courts","Bohemia Rd","","Hastings","","TN34 1ND","",201,"https://goo.gl/maps/NMZ43nC3w8N9jd6B8","Yes","Hastings",""
-1259,"SC228","SSCS Leeds","Newcastle Civil and Family Courts & Tribunal Centre","Barras Bridge","","Newcastle upon Tyne","","NE1 8QF","",101,"https://maps.google.com/maps?q=54.978706,-1.609995","Yes","Newcastle Civic",""
-1261,"SC338","SSCS Glasgow","Kilmarnock (Grand Hall)","Grand Hall","9 Green Street","East Ayrshire","Kilmarnock","KA1 3BN","",501,"https://goo.gl/maps/dV5v3JCXeu6BPBcB6","No","ZZ D U - Kilmarnock (Grand Hall)","Not active"
+000000,"1","SC001","SSCS Leeds","Barnsley SSCS Hearing Venue","Wellington House","","Wellington Street","Barnsley","S70 1WA","0223 391230",101,"https://www.google.com/maps/place/Wellington+House+Wellington+Street+Barnsley+S70+1WA/@53.550589,-1.480475","No","Barnsley","Not active"
```
Comment (MohammedHMCTS) [inline src/main/resources/reference-data/sscs-venues.csv:1]: How come you put the ID number as a string and never kept it as an number?

**Review by MohammedHMCTS — CHANGES_REQUESTED** [review]

File: src/main/resources/reference-data/sscs-venues.csv (hunk 1 above)
Comment (JamieHMCTS) [reply src/main/resources/reference-data/sscs-venues.csv]: It was requested that the ID number should be converted to a string. 
(The most up to date commit also converts the ePIMS ID number to a string)

**Review by MohammedHMCTS — APPROVED** [review]

**Review by dattipoeHMCTS — CHANGES_REQUESTED** [review]
Could you please build an RC version and ensure it plays nicely with sscs-tribunals-case-api? Can have a chat about it if needs be.

File: src/main/resources/reference-data/sscs-venues.csv [hunk 2]
```diff
@@ -1,266 +1,266 @@
...
-1257,"SC336","SSCS Bradford","Hastings Law Courts","Bohemia Rd","","Hastings","","TN34 1ND","",201,"https://goo.gl/maps/NMZ43nC3w8N9jd6B8","Yes","Hastings",""
-1259,"SC228","SSCS Leeds","Newcastle Civil and Family Courts & Tribunal Centre","Barras Bridge","","Newcastle upon Tyne","","NE1 8QF","",101,"https://maps.google.com/maps?q=54.978706,-1.609995","Yes","Newcastle Civic",""
-1261,"SC338","SSCS Glasgow","Kilmarnock (Grand Hall)","Grand Hall","9 Green Street","East Ayrshire","Kilmarnock","KA1 3BN","",501,"https://goo.gl/maps/dV5v3JCXeu6BPBcB6","No","ZZ D U - Kilmarnock (Grand Hall)","Not active"
+"1","SC001","SSCS Leeds","Barnsley SSCS Hearing Venue","Wellington House","","Wellington Street","Barnsley","S70 1WA","0223 391230",101,"https://www.google.com/maps/place/Wellington+House+Wellington+Street+Barnsley+S70+1WA/@53.550589,-1.480475","No","Barnsley","Not active","000000"
```
Comment (dattipoeHMCTS) [inline src/main/resources/reference-data/sscs-venues.csv:1]: Why are the first numbers now strings?

Comment (MohammedHMCTS) [reply src/main/resources/reference-data/sscs-venues.csv:1]: @JamieHMCTS:
It was requested that the ID number should be converted to a string.
(The most up to date commit also converts the ePIMS ID number to a string)

**Pull request closed without merging**